### ISSUE TITLE
run nightlybias even when 5min DARK is missing

### ISSUE THREAD

**Issue by sbailey** [desihub/desispec#1524]
Currently we couple the nightlybias calculation to the 5min DARK used for badcolumn masking, but there are some nights like 20210314 where we got the ZEROs (for nightlybias) but for some reason didn't get the 5min DARK (for bad columns), which currently results in the pipeline skipping making the nightlybias too.

From @akremin:
> It’s not common, but also happened more than once:
```
kremin@cori10:/global/cfs/cdirs/desi/spectro/redux/exptabs/exposure_tables> grep -L "calib dark 5min" ./202012/*.csv
./202012/exposure_table_20201208.csv
./202012/exposure_table_20201211.csv
./202012/exposure_table_20201213.csv
kremin@cori10:/global/cfs/cdirs/desi/spectro/redux/exptabs/exposure_tables> grep -L "calib dark 5min" ./2021??/*.csv
./202101/exposure_table_20210104.csv
./202101/exposure_table_20210106.csv
./202101/exposure_table_20210124.csv
./202101/exposure_table_20210125.csv
./202101/exposure_table_20210127.csv
./202102/exposure_table_20210225.csv
./202102/exposure_table_20210226.csv
./202103/exposure_table_20210310.csv
./202103/exposure_table_20210314.csv
./202103/exposure_table_20210330.csv
./202105/exposure_table_20210519.csv
./202106/exposure_table_20210629.csv
```

We should also be robust to this for the daily processing (though now the ZEROs and the DARK are the same script, so unlikely to miss just the DARK).

**Comment by sbailey** [comment]
When the DARK is missing, we can run `desi_proc --nightlybias ...` tied to a ZERO exposure, but we'll need to update `desispec.workflow.desi_proc_funcs.determine_resources` because the default cores and runtime are tied to a very fast single ZERO and it times out before finishing the nightlybias.  Currently the DARK resources are a hack assuming that you also want to do --nightlybias.

**Comment by sbailey** [comment]
Documenting a conversation with @akremin :

* keep --nightlybias + dark (for badcol) processing in a single job
* but similar to desi_proc on a science exposure, generate a batch script that does this in two srun calls:
  * one for --nightlybias with 8 ranks due to memory constraints
  * one for the actual dark with 30 ranks (faster)
* on nights without an actual dark, it would skip the second call
* from a processing table standpoint, rename this as jobdesc='ccdcalib' instead of 'dark'
* job script wrangling should add extra time if --nightlybias is included instead of hardcoding into determine_resources the expectation that darks (or zeros) need extra time and memory in case they are bundled with --nightlybias.

There may be other details that will become aparent when we try to implement this.

**Comment by sbailey** [comment]
fixed in #1553.
